### PR TITLE
assert_if revision

### PR DESCRIPTION
**Pull request by SamirDroubi**

### CODE REVIEW

File: src/exo/API_scheduling.py [hunk 1]
```diff
@@ -1942,30 +1942,25 @@ def lift_scope(proc, scope_cursor):
 
 
 @sched_op([IfCursorA, BoolA])
-def assert_if(proc, if_cursor, cond):
+def remove_if(proc, if_cursor, cond):
     """
```
Comment (yamaguchi1024) [inline src/exo/API_scheduling.py:1946]: Why does it take `cond` when it's not passed to `DoRemoveIf` ?

Comment (SamirDroubi) [reply src/exo/API_scheduling.py:1946]: Ahhh I forgot to deprecate the API parameter.

Comment (SamirDroubi) [reply src/exo/API_scheduling.py:1946]: Done

Comment (yamaguchi1024) [reply src/exo/API_scheduling.py:1946]: LGTM, I'll delegate it to @skeqiqevian to approve

**Review by skeqiqevian — APPROVED** [review]
Looks good to me!

**Pull request merged**